.github/workflows/test.yml

```diff
@@ -57,17 +57,25 @@ jobs:
             .github/actions/**
             .github/workflows/test.yml
 
+      - name: Root cargo related changes
+        id: cargo
+        uses: technote-space/get-diff-action@v6
+        with:
+          PATTERNS: |
+            Cargo.*
+            rust-toolchain
+
       - name: Rust related changes
         id: rust
         uses: technote-space/get-diff-action@v6
         with:
           PATTERNS: |
             pnpm-lock.yaml
             package.json
+            Cargo.**
```

Comment:
Is this still supposed to be here?

Reply:
Yeah because there are more general Rust jobs that should run if `Cargo.lock` changes.